drivers/src/hand_off.rs

```diff
@@ -1,11 +1,16 @@
+use core::mem::size_of;
```

Comment:
NIT: Move this after the license attribution.

Reply:
Fixed